java/code/src/com/redhat/rhn/frontend/xmlrpc/system/SystemHandler.java

```diff
@@ -678,6 +679,21 @@ public long scheduleChangeChannels(User loggedInUser, Integer serverId, String b
         return dr.toArray();
     }
 
+    /**
+     * Returns a list of empty system profiles visible to user (created by createSystemProfile).
+     *
+     * @param loggedInUser - the user
+     * @return array of empty system profiles
+     *
+     * @xmlrpc.doc Returns a list of empty system profiles visible to user (created by the createSystemProfile method).
+     * @xmlrpc.param #session_key()
```

Comment:
More doc needed what the return result is.

Example: listOutOfDateSystems()

```
     * @xmlrpc.returntype
     *          #array()
     *              $SystemOverviewSerializer
     *          #array_end()
```

You need to write a serializer for it.

Reply:
Thanks for this catch, fixing.